subsys/net/lib/lwm2m_client_utils/Kconfig

```diff
@@ -47,6 +47,26 @@ config LWM2M_CLIENT_UTILS_FIRMWARE_UPDATE_OBJ_SUPPORT
 	select FLASH
 	select MPU_ALLOW_FLASH_WRITE
 
+if LWM2M_CLIENT_UTILS_FIRMWARE_UPDATE_OBJ_SUPPORT
+
+config LWM2M_CLIENT_UTILS_FIRMWARE_INSTANCE_MODEM
+	int "Modem Firmware Object instance ID, highest in the range is not usable"
+	# Neither arithmetic operations are not possible when defining a range,
+	# nor excluding the values which equal to second symbol - or the first to that matter;
+	# it's impossibile to limit the range of the value to LWM2M_FIRMWARE_INSTANCE_COUNT-1.
+	range 0 LWM2M_FIRMWARE_INSTANCE_COUNT
+	default 0
```

Comment:
Move these explanations regarding the range to "help" section of Kconfig.

```
config LWM2M_CLIENT_UTILS_FIRMWARE_INSTANCE_MODEM
	int "Modem Firmware Object instance ID"
	range 0 LWM2M_FIRMWARE_INSTANCE_COUNT
	default 0
	help
          Instance ID must be in range from 0 to LWM2M_FIRMWARE_INSTANCE_COUNT-1.
          Highest in the range is not usable.
```

Reply:
Actually this is what you will see - the help section seems to get omitted. I placed the earlier section marked as comment to _help_ but it's not show anywhere.

![image](https://user-images.githubusercontent.com/33285290/132698932-710f9b8b-9c7d-4ad9-b33f-2867a44c9017.png)

Reply:
Am I missing something, as I got a build error from this config:
```
warning: the int symbol LWM2M_CLIENT_UTILS_FIRMWARE_INSTANCE_MODEM (defined at /home/seta/src/ncs/nrf/subsys/net/lib/lwm2m_client_utils/Kconfig:52) has a non-int range [0 (undefined), LWM2M_FIRMWARE_INSTANCE_COUNT (undefined)]

warning: the int symbol LWM2M_CLIENT_UTILS_FIRMWARE_INSTANCE_APP (defined at /home/seta/src/ncs/nrf/subsys/net/lib/lwm2m_client_utils/Kconfig:62) has a non-int range [0 (undefined), LWM2M_FIRMWARE_INSTANCE_COUNT (undefined)]

/home/seta/src/ncs/nrf/samples/nrf9160/lwm2m_client/prj.conf:36: warning: attempt to assign the value '2' to the undefined symbol LWM2M_FIRMWARE_INSTANCE_COUNT
```

Reply:
The help text is visible if you press `?`.
But I think it is OK to include the warning in both.

![2021-09-10 lwm2m_config](https://user-images.githubusercontent.com/3104794/132829930-313077fc-c02b-4f85-ade8-5a5d6dc729b9.png)